torch/_functorch/_aot_autograd/subclass_parametrization.py

```diff
@@ -79,11 +87,11 @@ def unwrap_tensor_subclass_parameters(module: torch.nn.Module) -> torch.nn.Modul
     becomes: {"parametrizations.p2.original0": torch.Tensor, "parametrizations.p2.original1": torch.Tensor}
 
     """
-    name_param: list[tuple[str, torch.nn.Parameter]] = list(
-        module.named_parameters(recurse=False)
-    )
-    for name, param in name_param:
-        if is_traceable_wrapper_subclass(param):
+    for name, tensor in itertools.chain(
+        list(module.named_parameters(recurse=False)),
```

Comment:
I see that this is pre-existing, but why do we set `recurse=False`? From the docs, it looks like this won't pick up any subclass params on inner submodules. We should probably fix that and add a test confirming that this was broken before

Reply:
Yeah i was just cargo culting lol, will take a closer look

Reply:
This is fine because:
1. Parameterizaiton API doesn't seem to work on FQN for params (bar.parameter)
2. This function is recursive and it does visit children modules.